app/server/models/ussd.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy.dialects.postgresql import JSON
+
+from server import db, sentry
+from server.models.utils import ModelBase
+
+
+class UssdMenu(ModelBase):
+    __tablename__ = 'ussd_menus'
+
+    name = db.Column(db.String, nullable=False, index=True, unique=True)
+    description = db.Column(db.String)
+    parent_id = db.Column(db.Integer)
+    """
+        TODO: change how we do i18n
+        keeping this as is from migration from grassroots economics
+        but this system is really bad since we always need 
+        all languages defined for all new menu items, no graceful fallback 
+    """
+    display_text_en = db.Column(db.String, nullable=False)
+    display_text_sw = db.Column(db.String, nullable=False)
+
+    @staticmethod
+    def find_by_name(name: str) -> "UssdMenu":
+        menus = UssdMenu.query.filter_by(name=name)
+        if menus.count() == 0:
+            sentry.captureMessage("No USSD Menu with name {}".format(name))
+            # should handle case if no invalid_request menu?
```

Comment:
yep just ran into this - fails bad atm. Could probably just have something generic.

Reply:
this should just not happen once we do the db seed for menus. all devs should run the seed as part of their setup so they at least have some data to work off of